### PR TITLE
Adds Discord bot messaging.

### PR DESCRIPTION
**Pull request by PJB3005**
Gets relayed to https://github.com/PJB3005/MoMMI with my web server.

### CODE REVIEW

File: code/game/gamemodes/game_mode.dm [hunk 1]
```diff
@@ -164,6 +164,7 @@
 		feedback_set("escaped_on_pod_5",escaped_on_pod_5)
 
 	send2mainirc("A round of [src.name] has ended - [surviving_total] survivors, [ghosts] ghosts.")
+	send2maindiscord("A round of **[name]** has ended - **[surviving_total]** survivors, **[ghosts]** ghosts.")
```
Comment (ComicIronic) [inline code/game/gamemodes/game_mode.dm:167]: You may as well fold these into one proc for the future so people don't leave one out - it won't kill IRC to get the extra formatting.

Comment (PJB3005) [reply code/game/gamemodes/game_mode.dm:167]: Uh, yes it will.

On IRC the formatting is complete clutter.

**Comment by clusterfack** [comment]
Config file is in the secret repo you should be able to modify it

**Comment by PJB3005** [comment]
Aha, will do.

**Comment by Kurfursten** [comment]
> Gets relayed to https://github.com/PJB3005/MoMMI

:-1: I don't think this will work, I've tried sending commands to MoMMIs before and they won't even open doors.

**Pull request merged**